README.md

```diff
@@ -25,11 +25,10 @@ Core features include:
 
 <!-- toc -->
 
-- [Talawa API](#talawa-api)
-  - [Talawa Components](#talawa-components)
-  - [Documentation](#documentation)
-  - [Installation](#installation)
-  - [Image Upload](#image-upload)
+- [Talawa Components](#talawa-components)
+- [Documentation](#documentation)
+- [Installation](#installation)
+- [Image Upload](#image-upload)
```

Comment:
No need to change this file

Reply:
Actually sir, I did not change this file. Maybe it was due to a previous PR that got reverted while I was working on this feature branch. Thank You.